docs/en/observability/synthetics-settings.asciidoc

```diff
@@ -0,0 +1,88 @@
+[[synthetics-settings]]
+= Adjust Synthetics settings
+
+++++
+<titleabbrev>Adjust settings</titleabbrev>
+++++
+
+There are several Synthetics settings you can adjust in {kib}.
+
+[discrete]
+[[synthetics-settings-alerting]]
+= Alerting
+
+Alerting enables you to detect complex conditions using *rules* across Observability apps
+and send a notification using *connectors*.
+
+When you create a new synthetic monitor, a new default synthetics rule will be applied.
+To edit the default rule:
+
+. Click *Alerts and rules* in the top bar.
+. Click *Monitor status rule* to open a panel where you can edit the rule's configuration.
+
```

Comment:
We've recently been [discussing this](https://github.com/elastic/kibana/issues/152430) and are looking to remove some of the configuration options in here.  It might be worth adding something like _"The automatically created `Synthetics internal alert` is intentionally pre configured (for instance you cannot change how often it checks the rule).  If you need some specific alerting behaviour, set up a different rule."_